app/src/marketplace/i18n/en.po

```diff
@@ -101,6 +101,15 @@ msgstr "Enter ETH address"
 msgid "editProductPage##setPrice##fixPrice"
 msgstr "Fix price in fiat for protection against shifts in the DATA price"
 
+msgid "editProductPage##sharedSecrets##addSecret"
+msgstr "Add shared secret"
+
+msgid "editProductPage##sharedSecrets##description"
+msgstr ""
+"Shared secrets are the access control for your product. "
+"Create, name and revoke shared secrets from here. "
+"For help with shared secrets, see the docs. "
```

Comment:
I haven't seen the designs, but we should probably be linking the docs here, as we do in the stream create - stream preview section.

Reply:
Yeah, we probably should. I don't think we have a link to provide here yet. Of course we could link to the root but I don't know if thats good or not :/